frontend/src/components/ProjectList/components/Dataset/common/style.ts

```diff
@@ -13,3 +13,23 @@ export const SmallColumn = styled.div`
   ${columnStyle}
   ${fontStyle}
 `;
+
+export const buttonStyle = css`
```

Comment:
consolidate button style for both "View" and "Download" here

Reply:
This is great.